Sources/KituraWebSocket/WebSocketError.swift

```diff
@@ -19,6 +19,9 @@ import Foundation
 /// An error enum used when throwing errors within KituraWebSocket.
 public enum WebSocketError: Error {
     
+    /// An invalid RSV was received in a WebSocket frame
+    case invalidRSV(UInt8)
```

Comment:
This is a breaking change so will need a major version (switches have to be exhaustive)

Reply:
Technically the RSV bits aren't part of the opcode so you did the right thing, but... ugh this is annoying.

Reply:
They kind of are with how we are currently treating them. Currently If you send an op code of 25 it will send a ping with rsv = 7

Reply:
Perhaps that's one of the other testcase failures!